### PR TITLE
fix(chips): error with undefined or null children

### LINKED ISSUE #880: ChipSet errors when rendered without children
The following error occurs if the `ChipSet` component renders without any children:

image

### PR DESCRIPTION
**Pull request by lucasecdb**
fixes #880

### CODE REVIEW

File: packages/chips/package.json [hunk 1]
```diff
@@ -22,9 +22,13 @@
...
     "classnames": "^2.2.6",
-    "react": "^16.4.2"
+    "react": "^16.4.2",
+    "react-is": "^16.8.6"
   },
   "publishConfig": {
     "access": "public"
+  },
+  "devDependencies": {
+    "@types/react-is": "^16.7.1"
```
Comment (moog16) [inline packages/chips/package.json]: we should also have @types/react here as well. I have an issue to add it to all components, but until we get to that let's add it here too.

Comment (lucasecdb) [reply packages/chips/package.json]: done!

Comment (lucasecdb) [reply packages/chips/package.json]: I added to all packages in #898

Comment (moog16) [reply packages/chips/package.json]: oh nice!

**Pull request merged**